### PR TITLE
MAT-6383 bug fix for saving QDM population Criteria

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-6383](https://jira.cms.gov/browse/MAT-6383)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx [hunk 1]
```diff
@@ -580,7 +580,7 @@ const MeasureGroups = (props: MeasureGroupProps) => {
   const orderingMeasureGroupObservations = (group: Group) => {
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: not your ticket but while you are here can we update the method name? we really need to work on naming conventions

**Review by sb-prateekkeerthi — DISMISSED** [review]

File: src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx (hunk 1 above)
Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: Any suggestions ? I am having hard time understanding what is even happening here.

Comment (sb-prateekkeerthi) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: Sorry I was the one who had given that name, so basically in that function we are ordering the measure observations such that always first denominatorObservation is stored and then numeratorObservation. this is important for QDM execution results display

Comment (adongare) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: No worries Prateek. Maybe `sortOberservations`? and one line documentation- sort observations to follow [denom observation, numer observation] order or something similar

**Review by adongare — APPROVED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]

**Pull request merged**